nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -471,7 +471,8 @@ def build_transformer_config(self) -> TransformerConfig:
 
         bias_dropout_fusion = self.cfg.get('bias_dropout_add_fusion', True)
 
-        apply_rope_fusion = self.cfg.get('apply_rope_fusion', True)
+        # @chcui default rope fusion to false until #8590 is closed.
+        apply_rope_fusion = self.cfg.get('apply_rope_fusion', False)
```

Comment:
should this be added to megatron_gpt_config.yaml?

Reply:
good idea, i'll do that